packages/yarnpkg-pnpify/sources/buildNodeModulesTree.ts

```diff
@@ -81,7 +89,7 @@ export const buildLocatorMap = (rootPath: PortablePath, nodeModulesTree: NodeMod
         map.set(val.locator, entry);
       }
 
-      entry.locations.push(ppath.relative(rootPath, location));
+      entry.locations.push(location);
```

Comment:
This change will result in more memory consumption by the locator map data structure. Why do you need it?

Reply:
The linker was returning relative paths to the project, which was causing issues when running `yarn install` from a subdirectory of the project (since they were accidentally interpreted against the cwd). Generally speaking, unless we hit actual memory overflows otherwise, I'd prefer to use absolute paths for as much of the pipeline as possible (defensive programming).

Reply:
I see, makes sense